### ISSUE TITLE
Unknown inspection: phpinspectionsea.

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#1247]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), v2.0.13 |
| Language level | PHP 7.0                        |

Current behaviour
--
error message: ```Unknown inspection: phpinspectionsea.```

code: 
```php
<?php

/**
 * @return null|string e.g. null, "0.00000", "3.42", ...
 */
function getPrice()
{
    // ...
    if (random_int(0, 1) === 1) {
        return null;
    }

    return "0.00000";
}

$price = getPrice();

/** @noinspection SuspiciousBinaryOperationInspection - bug is reported https://github.com/kalessil/phpinspectionsea/issues/1246 */
if (
    $price == null
    ||
    $price == 0
) {
    // ...
    var_dump($price);
}
```

Expected behaviour
--
"@noinspection" semms not to work with a comment? I don't know if this is the expected behaviour?

Environment details
---
PhpStorm 2018.3.5
Build #PS-183.5912.26, built on March 7, 2019
Licensed to Portable UTF-8 / Lars Moelleken
Subscription is active until February 28, 2020
For non-commercial open source development only.
JRE: 1.8.0_152-release-1343-b28 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.4.0-142-generic

**Comment by kalessil** [comment]
Fixed!
